gradle/publish.gradle

```diff
@@ -30,25 +30,6 @@ publishing {
             }
         }
     }
-
-
-    repositories {
-        if (version.version.isSignificant()) {
-            sonatypeMavenCentral {
-                credentials {
-                    username = project.findProperty('sonatype_username')
-                    password = project.findProperty('sonatype_password')
-                }
-            }
-        } else {
-            sonatypeSnapshots {
-                credentials {
-                    username = project.findProperty('sonatype_username')
-                    password = project.findProperty('sonatype_password')
-                }
-            }
-        }
```

Comment:
I'm guessing the new plugin does this automatically?

Reply:
It's in the main build.gradle now